### PR TITLE
Created a skill page and added a route

### LINKED ISSUE #262: Page: Create a page that will list all skills
Create a new page under `/admin/skills` which will fetch the list of skills from the API and display them.

> Note: Since there is no design for this page, be creative a bit :)

### PR DESCRIPTION
**Pull request by AnilMula**
# Description
This PR is to create a new skills page and adding  a route '/admin/skills'

Fixes #262 

# How to test?
Tested in local machine by running the project and giving the url in the browser http://localhost:3000/admin/skills

here is the test screenshot

image

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/App.js [hunk 1]
```diff
@@ -52,7 +53,9 @@ function App() {
         <Route exact path="/registration/success">
           <RegistrationSuccessPage />
         </Route>
-
+        <Route exact path="/admin/skills">
```
Comment (tpetrina) [inline src/client/App.js:56]: Should be `AuthenticatedRoute`.

File: src/client/containers/SkillsPage/SkillsPage.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import Loader from '../../components/Loader/Loader';
+import { AppHeader } from '../../components/Appheader/AppHeader.component';
+import Footer from '../../components/footer/footer';
+import Label from '../../components/Label/Tags/Label';
+
+export default function SkillsPage() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [skills, setSkills] = useState([]);
+  useEffect(() => {
+    axios.get('/api/skills').then((response) => {
```
Comment (tpetrina) [inline src/client/containers/SkillsPage/SkillsPage.js:13]: Use `fetch` from #255.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Great start, just a little bit!

**Review by tpetrina — DISMISSED** [review]
Good stuff.

**Review by tpetrina — APPROVED** [review]
Looks good

**Review by Elia-Git — APPROVED** [review]
Good job!

**Pull request merged**